src/http/request_handler.rs

```diff
@@ -113,6 +114,26 @@ pub async fn handle_request<T: DeserializeOwned + GraphQLRequestLike, Http: Http
   state: Arc<AppContext<Http, Env>>,
```

Comment:
we should pass the app context from here instead.

Reply:
wdym?